app/styles/utils/_layoututils.scss

```diff
@@ -20,6 +20,7 @@ $header-height: 70px;
 $header-mobile-height: 50px;
 
 // Z-Index values
+$z-index-button-bar: 1;
```

Comment:
This comment might not be relevant if you pull the button bar out to be a sibling of the Layout component, but I wanted to leave this comment in case you didn't want to do that in this PR and/or for educational purposes.

We probably shouldn't be defining this constant in this file because the element that you are applying this to is not in the same [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) as the others, and therefore has no relation to the other z-index values described below. z-indexes aren't actually global; they're only relative to other elements within the same stacking context, and, as the name implies, there can be multiple stacking contexts within a page. As a concrete example, even if you set this constant to a very large z-index value, it would still stack below the `$z-index-modal: 5` constant, since the entire stacking context is still below a z-index of 5.

My mental heuristic for knowing whether an element is in a different stacking context is whether any ancestor of the element has been taken out of the [normal flow](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Normal_Flow) of the layout engine. Absolutely/relatively positioning an element will take that element out of normal flow, and flexbox/grid will take the children of an element out of normal flow.

Reply:
Since we decided to not float the button bar, I've removed the z-index const. But thank you for the explanation nonetheless!